e2e-test/fluxd.yaml

```diff
@@ -0,0 +1,166 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: flux-kube-config
+  namespace: default
+data:
+  config: |
+    apiVersion: v1
+    clusters: []
+    contexts:
+    - context:
+        cluster: ""
+        namespace: default
+        user: ""
+      name: default
+    current-context: default
+    kind: Config
+    preferences: {}
+    users: []
+---
+apiVersion: v1
+data:
+  identity: LS0tLS1CRUdJTiBPUEVOU1NIIFBSSVZBVEUgS0VZLS0tLS0KYjNCbGJuTnphQzFyWlhrdGRqRUFBQUFBQkc1dmJtVUFBQUFFYm05dVpRQUFBQUFBQUFBQkFBQUJGd0FBQUFkemMyZ3RjbgpOaEFBQUFBd0VBQVFBQUFRRUF5Q0JEcHZXc29xY1JoRXIvcWcvZmduS3NIaTljT1Boa1BvWXRnN2dZU0xGdUJ6Rks0RDV6CkZOZjNIbXR1K0trM3N4RU41QlVqK3RWY2k1YlkreHlQWEdUNjlqS01rOHJKcUErRDJrNmo1U1A4aUY4NG5ZQk82NGNIakYKWkEwT3NhM1RiQ21QSEt4S0lma1ZVYTZLbHBCZUc2TEwwcjlEcnNBcDNnK2dqbE5DTnZrWnRpbUVNVTIxOWhWQkxLM1VBcQpadzgxZU5HejNCRmhJdHM5aXI5SWRiLzZycm80YksreDRnMzdFOFBjb3cxYWhRRGJQM2loVThGM3N2TlZjd3RnMmVuZnVxCmkrNG0zdzE2VEE4TWljSmNwYWJOV1lqNnNneGwxbVB5bW85YzNjNDk0Z3puK0lUNHRWWGpwVnRWb0Y4MG9JZjlEYjBqeTQKTlpSOGRSSklCUUFBQTloejNLd1hjOXlzRndBQUFBZHpjMmd0Y25OaEFBQUJBUURJSUVPbTlheWlweEdFU3YrcUQ5K0NjcQp3ZUwxdzQrR1EraGkyRHVCaElzVzRITVVyZ1BuTVUxL2NlYTI3NHFUZXpFUTNrRlNQNjFWeUxsdGo3SEk5Y1pQcjJNb3lUCnlzbW9ENFBhVHFQbEkveUlYemlkZ0U3cmh3ZU1Wa0RRNnhyZE5zS1k4Y3JFb2grUlZScm9xV2tGNGJvc3ZTdjBPdXdDbmUKRDZDT1UwSTIrUm0yS1lReFRiWDJGVUVzcmRRQ3BuRHpWNDBiUGNFV0VpMnoyS3YwaDF2L3F1dWpoc3I3SGlEZnNUdzl5agpEVnFGQU5zL2VLRlR3WGV5ODFWekMyRFo2ZCs2cUw3aWJmRFhwTUR3eUp3bHlscHMxWmlQcXlER1hXWS9LYWoxemR6ajNpCkRPZjRoUGkxVmVPbFcxV2dYelNnaC8wTnZTUExnMWxIeDFFa2dGQUFBQUF3RUFBUUFBQVFCWTVvUkhFZEdydTUyN2IvbWMKMlVwSHo1c0lHYS9WR3ZxbEQ4UCtSSFI5TGx5YlUxbFl2a1V2R0RVTnpWblJtcEpFdUttOERXd2JURmE5SjdpaDdIRCs4VgpqT3IydmxaMnNuZmlUcmdJMnNCODF4NHlMcVRJcENkSXgwMmdnWEJxbTdaVTBTMWk1bUpVTHlzNlN0Z0R4SVFac0xyc2xzCnlCUDVvWnJFVjlsQnpFYSttUXF3NWlodG1JUnlIWDNadFZEYkQ5VUJKZ01MbVAvY3Q1YlJ4RjhXTFU5R3hWd2xob1Q0aFEKQ2pBRlJBUlpBeHczTmlUUVVReDhYUDgydnhjZmdwTDJrc1k2dFFwSWdNREovNjJrYjNSaVJocjBGTGlKaGw2WkNCd2hJcApjK05UVzVhcXNTRWF0OURRRWRYcmNuK2hzTEtOZGVoUHUxeGxmS3IvYzRGdEFBQUFnUURtcGwrMU9GdXFwaTQrYnI0d3EyCmtkY1AvRk9jQlZuWXpwODE4eG1TQzNnMlc5YXlrb2FCem9jaTFNaXBoQVRWbTB1RDBzWHN2YXEzYW9ESWhBQ2FNL2t6bjkKcmNGb0JqeDFNUXYxVWUrWEJHWWN6S2V2OHFzR1ZYaWpwSnhIRGdNZkhTWmJHcEVzNmFhanpyNlJjeW1YR20wdVI3UGxFNApWQXU0L2JtdXljR2dBQUFJRUE3UWl3WHdaQzlnUWY1NVlpUDNsM1dDU3JBUFJ5Qjd2WXQxQW1BeWplRDM5K2lNcGNrVDViClRLWUo5ckcraWsvWHZNSVJKM1cyYW9LREMzd25STWhOMGdvcHpWTU9ZMVlnV2Vid0ZJdWwvcjRTZW5Jb0l3TG5Vcm1TYjEKK0YrUzVZRWRMUHBibGZRa1kxdDlzbUNqUzFIcTVVVkZ3VGw0KzdpK3Z3OS8zNVRjc0FBQUNCQU5namtXelBMNHg4cHB5dwptY2YwMkxBMVVZbE5idEpSbGk5aktTRE1wOGtxNkJWSjh4SGwwcWVHNWwxYWkvaTAvaWxWd3ZYb1JKYnBuSThPUFlJNW43Cm0vZWtSd3doamtWWnBoMDNEbExtckc4dkZVZUNtOFZuT2tNQk00YmVZSnd1ZGhBOWMxSzdUcHJkNkRaaUFzNGxMMXBDQTUKeWZkWkJ2UVhRdC9qYXdkdkFBQUFIa1Z0YVd4QVJXMXBiSE10VFdGalFtOXZheTFRY204dE1pNXNiMk5oYkFFQ0F3UT0KLS0tLS1FTkQgT1BFTlNTSCBQUklWQVRFIEtFWS0tLS0tCg==
```

Comment:
Is this needed and in that case, where does it come from?

Reply:
It's needed in the current setup, just because it isn't optional. It is just generated on my machine. Will write a comment on it.

Reply:
Good enough?